FAQ cleanup (#486)
kiali/kiali.io: content/en/docs/FAQ/distributed-tracing.md

{"data":null,"total":0,"limit":0,"offset":0,"errors":[{"code":400,"msg":"parameter 'service' is required"}]}
```

> If you see some returning JSON as in the above example, that should be the URL that you must configure.

Reviewer: ```suggestion
If you see some returned JSON, as in the above example, that is the URL to use in the configuration.
```
